src/routes/reference/basic-reactivity/create-resource.mdx

```diff
@@ -97,7 +97,7 @@ You can now check whether a Resource is `unresolved`, `pending`, `ready`, `refre
 | `refreshing` | Yes            | Yes     | No        |
 | `error`      | No             | No      | Yes       |
 
-2. When server rendering resources especially when fetching when embedding Solid in other system that fetch before render, you might want to initiate the resource with this prefetched value instead of fetching again and having the resource serialize it in it's own state.
+2. When server rendering resources especially when fetching when embedding Solid in other systems that fetch data before rendering, you might want to initiate the resource with this prefetched value instead of fetching again and having the resource serialize it in it's own state.
```

Comment:
```suggestion
2. When server-rendering resources, especially when embedding Solid in other systems that fetch data before rendering, you might want to initialize the resource with this prefetched value instead of fetching again and having the resource serialize it in its own state.
```

(Your changes here are good, but reading the sentence, I spotted some other fixes that would help.)